pages/reports/[id].vue

```diff
@@ -16,7 +16,7 @@ const selected = usePackageInUrl({ report });
 
 const criteria = computed(() => ({
   ...MISSING_CRITERIA_KEYS,
-  ...report?.value?.criteria,
+  ...report.value.criteria,
```

Comment:
**question:** If `report.value.criteria` is `undefined` (like it would be with the placeholder `{}`) is this going to throw, since we can't spread `undefined`?